Mathlib/Algebra/Category/ModuleCat/ChangeOfRings.lean

```diff
@@ -207,6 +207,7 @@ variable (M : Type v) [AddCommMonoid M] [Module R M]
 -- mathport name: «expr ⊗ₜ[ , ] »
 -- This notation is necessary because we need to reason about `s ⊗ₜ m` where `s : S` and `m : M`;
 -- without this notation, one need to work with `s : (restrictScalars f).obj ⟨S⟩`.
+/-- Tensor multiplication using a ring morphism. -/
```

Comment:
Why are we adding a docstring in this PR?

Reply:
I gave two reasons on Zulip :) 

Removed.